### DISCUSSION TITLE
Profile.DefaultMemberConfig is not accessible in v11

### DISCUSSION THREAD

**Discussion by maxkoshevoi** [LuckyPennySoftware/AutoMapper#3977] [NA]
This code no longer works: https://github.com/AutoMapper/AutoMapper/issues/1269#issuecomment-292823135 (while it worked in v10)
Is there a new way to clear all default configs to use only explicit mappings?

**Comment by maxkoshevoi** [comment]
Figured it out:

```cs
using AutoMapper.Internal;

((IProfileExpressionInternal)this).DefaultMemberConfig
```

**Reply by mikebouck** [reply]
I had this same issue as well although it seems like explicitly casting to an "internal" interface has code smell.  If this is the intended method to gain access to the profile's MemberConfig then it should, at the very least, be documented as such.

**Reply by lbargaoanu** [reply]
https://docs.automapper.org/en/latest/11.0-Upgrade-Guide.html#configuration-performance

**Reply by mikebouck** [reply]
Thanks!  Just be aware that there are consumers using custom name to member mappers in Profile-derived classes.
